tests/exchange/test_exchange.py

```diff
@@ -3909,3 +3972,70 @@ def test__amount_to_contracts(
     assert result_size == param_size
     result_amount = exchange._contracts_to_amount(pair, param_size)
     assert result_amount == param_amount
+
+
+@pytest.mark.parametrize('exchange_name,open_rate,is_short,leverage,trading_mode,collateral', [
+    # Bittrex
+    ('bittrex', 2.0, False, 3.0, 'spot', None),
+    ('bittrex', 2.0, False, 1.0, 'spot', 'cross'),
+    ('bittrex', 2.0, True, 3.0, 'spot', 'isolated'),
+    # Binance
+    ('binance', 2.0, False, 3.0, 'spot', None),
+    ('binance', 2.0, False, 1.0, 'spot', 'cross'),
+    ('binance', 2.0, True, 3.0, 'spot', 'isolated'),
+])
+def test_liquidation_price_is_none(
+    mocker,
+    default_conf,
+    exchange_name,
+    open_rate,
+    is_short,
+    leverage,
+    trading_mode,
+    collateral
+):
+    default_conf['trading_mode'] = trading_mode
+    default_conf['collateral'] = collateral
+    exchange = get_patched_exchange(mocker, default_conf, id=exchange_name)
+    assert exchange.get_liquidation_price(
+        pair='DOGE/USDT',
+        open_rate=open_rate,
+        is_short=is_short,
+        position=71200.81144,
+        wallet_balance=-56354.57,
+        mm_ex_1=0.10,
+        upnl_ex_1=0.0
+    ) is None
+
+
+@pytest.mark.parametrize(
+    'exchange_name, is_short, leverage, trading_mode, collateral, wallet_balance, '
+    'mm_ex_1, upnl_ex_1, maintenance_amt, position, open_rate, '
+    'mm_ratio, expected',
+    [
+        ("binance", False, 1, 'futures', 'isolated', 1535443.01, 0.0,
+         0.0, 135365.00, 3683.979, 1456.84, 0.10, 1114.78),
+        ("binance", False, 1, 'futures', 'isolated', 1535443.01, 0.0,
+         0.0, 16300.000, 109.488, 32481.980, 0.025, 18778.73),
+        ("binance", False, 1, 'futures', 'cross', 1535443.01, 71200.81144,
+         -56354.57, 135365.00, 3683.979, 1456.84, 0.10, 1153.26),
+        ("binance", False, 1, 'futures', 'cross', 1535443.01, 356512.508,
+         -448192.89, 16300.000, 109.488, 32481.980, 0.025, 26316.89)
+    ])
+def test_liquidation_price(
+    mocker, default_conf, exchange_name, open_rate, is_short, leverage, trading_mode,
```

Comment:
I noticed that `leverage` is not relevant for these formulaes ... is that correct/intentional? (maybe we can remove it from tests then as well)